src/runtime/generators.js

```diff
@@ -52,6 +52,31 @@
                       state);
   }
 
+  // The following unique object serves as a non-catchable exception raised by
+  // the implementation of Generator.prototype.return
+  var SENTINEL = new Object();
+  
+  function GeneratorWrapper(iterator) {
```

Comment:
Or maybe make the API a non constructor

```js
$tracuerRuntime.wrapYieldStar(expr)
```

The reason is that these runtime functions stay more conistent if they are all function calls.